en/docs/learn/configuring-sms-otp.md

```diff
@@ -1,210 +1,160 @@
-			</tr>
-			<tr>
-				<td>BackupCode</td>
-				<td>This is to use define whether to use a backup code instead of the actual SMS code or not.</td>
-			</tr>
-			<tr>
-				<td>SMSOTPEnableByUserClaim</td>
-				<td>This facilitates disabling the 'SMS OTP disabling byuser' functionality. The value can be <code>true</code> or <code>false</code>. If the value is set to <code>true</code>, the user can enable and disable the SMS OTP according to what the admin selects (SMSOTPMandatory parameter value).</td>
-			</tr>
-			<tr>
-				<td>SMSOTPMandatory</td>
-				<td>If the value is true, the second step will be enabled by the admin. The user cannot be authenticated without SMS OTP authentication. This parameter is used for both super tenant and tenant in the configuration. The value can be <code>true</code> or <code>false</code>.</td>
-			</tr>
-			<tr>
-				<td>CaptureAndUpdateMobileNumber</td>
-				<td>When <code>SMSOTPMandatory</code> is set to <code>true</code> and the user forgets to update the mobile number in a specific user profile where this property is set to <code>true</code>, the user can update a mobile claim with value in authentication time (If it is first login) and get the mobile number from the user's profile to send OTP. This update functionality will happen in the first login only. Once the user updates the mobile number, in the next login, the mobile number will take the updated mobile number from that specific user profile.</td>
-			</tr>
-			<tr>
-				<td>SendOTPDirectlyToMobile</td>
-				<td>When <code>SMSOTPMandatory</code> is set to <code>true</code> and the user does not exist in the user store and if the admin sets <code>SendOTPDirectlyToMobile</code> to <code>true</code>, the user can enter the mobile number in authentication time in a mobile number request page; the OTP will be directly sent to that mobile number.</td>
-			</tr>
-			<tr>
-				<td>redirectToMultiOptionPageOnFailure</td>
-				<td></td>
-		</tbody>
-	</table>
-
-	!!! note
-
-		The SMS provider is the entity that is used to send the SMS. The SMSOTP connector has been configured such that it can be used with most types of SMS APIs. Some use the GET method with the client secret and API Key encoded in the URL (e.g., Nexmo), while some may use the POST method when sending the values in the headers and the message and telephone number in the payload (e.g., Clickatell). Note that this could change significantly between different SMS providers. The configuration of the connector in the identity provider would also change based on this.
+    -   To ensure you get the full understanding of configuring SMS OTP with WSO2 Identity Server, 
+        the sample travelocity application is used in this use case. Therefore, make sure to 
+        [download the travelocity 
+        application](https://maven.wso2.org/nexus/content/groups/public/org/wso2/is/org.wso2.sample.is.sso.agent/5.9.0/).
```

Comment:
Let's point to direct download link https://maven.wso2.org/nexus/content/groups/public/org/wso2/is/org.wso2.sample.is.sso.agent/5.9.0/org.wso2.sample.is.sso.agent-5.9.0.war

Reply:
Let's upload the renamed travelocity.war into https://github.com/wso2/samples-is/releases/tag/v4.1.0 and poind download link to there

Reply:
+1 for the suggestion.
@Buddhimah is making a new doc on configuring an SP and deploying travelocity. After that, we need to change the content of this doc according to it.

Reply:
Fixed: e77f140d692f363cc6f763679385b3e1699216c2